dockyard/common/container/container.py

```diff
@@ -0,0 +1,162 @@
+from dockyard.common import utils, link
+
+import json
+from pecan.core import redirect
+
+class Container(object):
+    def __init__(self):
+        pass
+
+    def list(self, name_id=None, query_params=None):
+        if name_id:
+            url = ('/containers/%s/json' % (name_id))
+        else:
+            url = '/containers/json'
+
+        if query_params:
+            url = ('%s?%s' % (url, query_params))
+
+        return utils.dispatch_get_request(url, 'http').data
+
+    def changes(self, name_id=None):
+        url = (('/containers/%s/changes') % (name_id))
+        return utils.dispatch_get_request(url, 'http').data
+
+    def resize(self, _id, **kwargs):
+        url = (('/containers/%s/changes') % (_id))
+
+        if kwargs:
+            query = link.make_query_url(kwargs)
+            url = ('%s?%s' % (url, query ))
+
+        return utils.dispatch_get_request(url, 'http').data
```

Comment:
make sure that this 'http' is being passed, is it really used somewhere? is it really required?